Feature/fmt9 (#1324)
inviwo/inviwo: CHANGELOG.md

Here we document changes that affect the public API or changes that needs to be communicated to other developers. 

## 2022-08-15 FMT Version 9.0.0

> The fmt library was updated to the recent version 9.0.0. There are major breaking change in the update with respect to using ostream operators and fmt. Previosly you could just include `fmt/ostream.h` and any type that was streamable was not also printable with fmt. That behavious as removed and now you either as to wrapp the object with `fmt::streamed(x)` or add a specialization of fmt::formatter for the type. Most core types in inviwo that used std::ostream operators has been updated with fmt formatters. 

Reviewer: ```suggestion
The fmt library was updated to the recent version 9.0.0. There are major breaking changes in the update with respect to using ostream operators and fmt. Previously you could just include `fmt/ostream.h` and any type that was streamable was not also printable with fmt. That behavior was removed and now you either have to wrap the object with `fmt::streamed(x)` or add a specialization of fmt::formatter for the type. Most core types in Inviwo that used std::ostream operators have been updated with fmt formatters. 
```
